contracts/CapabilityProxy.cdc

```diff
@@ -90,11 +90,14 @@ pub contract CapabilityProxy {
             } else {
                 self.privateCapabilities.insert(key: cap.getType(), cap)
             }
+            emit ProxyUpdated(id: self.uuid, capabilityType: cap.getType(), isPublic: isPublic, active: true)
         }
 
         pub fun removeCapability(cap: Capability) {
-            self.publicCapabilities.remove(key: cap.getType())
+            let removedPublic = self.publicCapabilities.remove(key: cap.getType())
             self.privateCapabilities.remove(key: cap.getType())
+
+            emit ProxyUpdated(id: self.uuid, capabilityType: cap.getType(), isPublic: removedPublic != nil, active: false)
```

Comment:
Leaving this suggestion in case we want to handle the case that a proxy capability exists in both the private and public buckets

```suggestion
            if let removedPublic = self.publicCapabilities.remove(key: cap.getType()) {
                emit ProxyUpdated(id: self.uuid, capabilityType: cap.getType(), isPublic: true, active: false)
            }
            
            if let removedPrivate = self.privateCapabilities.remove(key: cap.getType()) {
                emit ProxyUpdated(id: self.uuid, capabilityType: cap.getType(), isPublic: false, active: false)
            }
```